### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by stonerl** [italomandara/CXPatcher#120] [General]
Is there a CXPatcher Discord?

**Comment by italomandara** [comment]
There's a channel in AppleGamingWiki's Discord

**Reply by stonerl** [reply]
I see, do have to be invited to join the channel? ATM, I only see rules and #support.

**Reply by italomandara** [reply]
I can see people that just joined messaging in cxpatcher's channel maybe check the rules probably you need to accept some disclaimer there? or just ask in #support

**Reply by italomandara** [reply]
Can you just leave the discord and come back?
You’re supposed to get a screen initially that explains the process
you do one or two things and then get access to all the channels
